CHANGELOG.rst

```diff
@@ -1,11 +1,16 @@
 lib_i2c change log
 ==================
 
-UNRELEASED
-----------
+6.3.0
+-----
+
   * REMOVED: Support for XS1 - Please design with xcore.ai for new projects
   * CHANGED: Build examples and tests using XCommon CMake instead of XCommon
 
+  * Changes to dependencies:
+
+    - lib_xassert: 4.2.0 -> 4.3.0
```

Comment:
Seems to be missing change to lib_logging 3.2.0 -> 3.3.0?

Reply:
ah ignore that, its the examples only

Reply:
Yeah, the changelog check only suggested xassert update